### PR TITLE
Add a paragraph about reviewers and review latency in Getting Started chapter

### LINKED ISSUE #837: Mention how long it takes to get code reviewed
The Getting Started guide mentions how long it can take for bors to merge PRs, but it does not mention how long it usually takes until someone reviews a PR. It would be helpful for new contributors to see an estimation of how long it will take for someone to review their PR and after how long it is appropriate to ping. These numbers probably vary person-to-person, but it would be helpful if there were a rough estimate.

### PR DESCRIPTION
**Pull request by LeSeulArtichaut**
cc @camelid @Dylan-DPC

Closes #837.

### CODE REVIEW

File: src/getting-started.md
Comment (camelid) [inline src/getting-started.md:384]:
```suggestion
To try to move PRs forward, the Triage WG regularly goes through all PRs that
```

File: src/getting-started.md
Comment (camelid) [inline src/getting-started.md:388]: I think it looks better without the ellipsis:

```suggestion
on vacation, etc.
```

**Review by camelid — APPROVED** [review]
This looks good to me! Just a couple nits :)

**Review by tshepang — APPROVED** [review]

File: src/getting-started.md
Comment (LeSeulArtichaut) [reply src/getting-started.md]: Hehe, in French I think we always put an ellipsis after etc 😁
I'll change it though, thanks for the suggestion

**Comment by camelid** [comment]
It looks like there's something wrong with CI (the blank lines are because Travis doesn't copy very well):

```
$ git checkout -b ci

Switched to a new branch 'ci'

The command "git checkout -b ci" exited with 0.

0.52s$ git rebase origin/master

First, rewinding head to replay your work on top of it...

Applying: add `mdbook init`

Using index info to reconstruct a base tree...

Falling back to patching base and 3-way merge...

CONFLICT (add/add): Merge conflict in src/SUMMARY.md

Auto-merging src/SUMMARY.md

CONFLICT (add/add): Merge conflict in .gitignore

Auto-merging .gitignore

error: Failed to merge in the changes.

hint: Use 'git am --show-current-patch' to see the failed patch

Patch failed at 0001 add `mdbook init`

Resolve all conflicts manually, mark them as resolved with

"git add/rm <conflicted_files>", then run "git rebase --continue".

You can instead skip this commit: run "git rebase --skip".

To abort and get back to the state before "git rebase", run "git rebase --abort".

The command "git rebase origin/master" exited with 1.

0.01s$ git log --oneline | head -n 10

6159dde Fix instructions for squashing

fdd5946 add the last missing code highlight to t-compiler in the docs

c58ff8d Fix incorrect windows link

8aaf8ed Add windows instructions

4a6fd11 Recommend setting up `check_line_lengths` as a pre-commit script

a8b7e92 Fix typo

c26ab70 Incorporated suggestions into git page and edit for more brevity.

be11a52 Edit using git page for brevity and clarity.

328bd35 Add a section on using git.

5ee9d96 --verbose is useful when not compiling and when compiling (#901)

The command "git log --oneline | head -n 10" exited with 0.

$ mdbook build

2020-10-03 18:49:50 [ERROR] (mdbook::utils): Error: Summary parsing failed

2020-10-03 18:49:50 [ERROR] (mdbook::utils): 	Caused By: There was an error parsing the suffix chapters

2020-10-03 18:49:50 [ERROR] (mdbook::utils): 	Caused By: failed to parse SUMMARY.md line 166, column 1: Suffix chapters cannot be followed by a list

The command "mdbook build" exited with 101.
```

**Comment by camelid** [comment]
But I'm pretty sure it's unrelated to this PR. Are we waiting on review from @Dylan-DPC?

File: src/getting-started.md
Comment (Dylan-DPC) [inline src/getting-started.md:386]: 15 days is our current policy or you can say "2 weeks"

Comment (LeSeulArtichaut) [reply src/getting-started.md]: Changed in 650a5b8, if you think that's correct we can merge this

**Comment by camelid** [comment]
Maybe squash so there's less noise in the Git history?

**Comment by LeSeulArtichaut** [comment]
> Maybe squash so there's less noise in the Git history?

I think we always squash-merge PRs here. We're not in `rust-lang/rust` ;)

**Review by mark-i-m — APPROVED** [review]
Thanks :)

**Pull request merged**
